src/utils/OpenSwathWorkflow.cpp

```diff
@@ -682,7 +684,10 @@ class TOPPOpenSwathWorkflow
 
     String readoptions = getStringOption_("readOptions");
     String mz_correction_function = getStringOption_("mz_correction_function");
-    String tmp = getStringOption_("tempDirectory");
+    
+    // make sure tmp is a directory with proper separator at the end (downstream methods simply do path + filename)
+    // (do not use QDir::separator(), since its platform specific (/ or \) while absolutePath() will always use '/')
+    String tmp_dir = String(QDir(getStringOption_("tempDirectory").c_str()).absolutePath()).ensureLastChar('/');
```

Comment:
before it was possible to specify filename and directory, but now this is not possible any more, leading to the test failures. This was a good idea if you wanted to analyze multiple runs without worrying about them overwriting each others files.